modules/hostap/src/supp_api.c

```diff
@@ -568,7 +568,8 @@ static int wpas_add_and_config_network(struct wpa_supplicant *wpa_s,
 				}
 			}
 
-			if (!wpa_cli_cmd_v("set_network %d key_mgmt SAE", resp.network_id)) {
+			if (!wpa_cli_cmd_v("set_network %d key_mgmt SAE%s", resp.network_id,
+			    params->ft_used ? " FT-SAE" : "")) {
```

Comment:
Please align the params to `"set_network...`

Reply:
Updated.